content/en/infrastructure/livecontainers.md

```diff
@@ -13,73 +13,100 @@ further_reading:
-
-## Installation
-
-Follow the [Docker][6] or [Kubernetes][7] Agent installation instructions. Enable the Process Agent to populate your Live Containers view. Container metrics are available without additional configuration after installation.
-
-**Kubernetes Resources for Live Containers requires installation of**:
-
-* [Datadog Agent][8] version 7.21.1 (or above)
-* [Datadog Cluster Agent][9] 1.9.0 (or above)
+## Configuration
 
 ### Kubernetes Resources
 
-To enable Kubernetes Resources for Live Containers, follow the [Helm instructions][10] and add the following changes to your `values.yaml` file:
-
-{{< code-block lang="yaml" filename="values.yaml" >}}
-datadog:
-  ...
-  processAgent:
-    enabled: true
-  ...
-  orchestratorExplorer:
-    enabled: true
-...
-clusterAgent:
-  enabled: true
-  image:
-    repository: datadog/cluster-agent
-    tag: latest
-    pullPolicy: Always
-...
-agents:
-  image:
-    repository: datadog/agent
-    tag: latest
-    pullPolicy: Always
-...
-{{< /code-block >}}
-
-In cases where the Agent is not able to automatically detect the Kubernetes cluster name, you must set it in `values.yaml`:
-
-{{< code-block lang="yaml" filename="values.yaml" >}}
-datadog:
-   ...
-   clusterName: <PLACEHOLDER>
-   ...
-{{< /code-block >}}
-
-**Note**: The cluster name must be 80-characters or less.
-
-On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the Agent and the cluster Agent don't have access to the cloud metadata APIs, or the cluster name is longer than 80 characters.
+The Agent and [Cluster Agent][5] can be configured to retrieve Kubernetes resources for [Live Containers][6]. This feature allows you to monitor the state of pods, deployments and other Kubernetes concepts in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+
+Kubernetes resources for Live Containers requires [Agent version >= 7.21.1][7] prior to the configurations below.
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+If you are using the official [Datadog Helm Chart][1]:
+
+- Use chart version 2.4.5 or above
+- Set `datadog.orchestratorExplorer.enabled` to true in [values.yaml][2]
+- Deploy a new release
+
+
+[1]: https://github.com/DataDog/helm-charts
+[2]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "DaemonSet" %}}
+
+To configure a DaemonSet, [Cluster Agent][1] version >= 1.9.0 is required. The Cluster Agent must be running, and the Agent must be able to communicate with it. See the [Cluster Agent Setup documentation][2] for configuration.
+
+1. Set the Cluster Agent container with the following environment variable:
+
+      ```
+      - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+        value: "true"
+      ```
+
+2. Set the Cluster Agent ClusterRole with the following RBAC permissions:
```

Comment:
I don't know if it's clear enough that the clusterrole needs to be updated with these additional permissions, but they might also need more for other features. It's not an extensive list. If you think it's clear let's keep it that way, otherwise maybe update with "Make sure these permissions are included in the Cluster Agent ClusterRole"

Reply:
I think this is descriptive enough as-is. 👍